include/UnifiedCvo/utils/CvoPointCloud.hpp

```diff
@@ -170,14 +181,16 @@ namespace cvo {
     int num_classes_;
     int feature_dimensions_;
     int num_geometric_types_;
-    
+
+    std::vector<cvo::CvoPoint> points_;
     //std::vector<Eigen::Vector3f, Eigen::aligned_allocator<Eigen::Vector3f>> positions_;  // points position. x,y,z
     std::vector<Eigen::Vector3f> positions_; 
     //Eigen::Matrix<float, Eigen::Dynamic, 3> positions_;
     Eigen::MatrixXf features_;
     //Eigen::Matrix<float, Eigen::Dynamic, Eigen::Dynamic> features_;   // rgb, gradient in [0,1]
     //std::vector<Eigen::Matrix<float, 1, Eigen::Dynamic>> features_;   // rgb, gradient in [0,1]
     //Eigen::Matrix<float, Eigen::Dynamic, Eigen::Dynamic> normals_;  // surface normals
+    //TODO: fix labels here
```

Comment:
The posiitons_, features_, and labels_ can be removed in the future

Reply:
I have checked that there were no calls to these variables in cvocloudpoint.cpp and cvopoint.cpp in this version. I will remove them in the next submit